test/MOIWrapper.jl

```diff
@@ -14,17 +14,19 @@ MOIU.@model(SCSModelData,
             (MOI.ScalarAffineFunction,),
             (MOI.VectorOfVariables,),
             (MOI.VectorAffineFunction,))
-const optimizer = MOIU.CachingOptimizer(SCSModelData{Float64}(), SCS.Optimizer())
+for T in [SCS.Direct, SCS.Indirect]
+    optimizer = MOIU.CachingOptimizer(SCSModelData{Float64}(), SCS.Optimizer(linear_solver=T))
 
-# linear9test needs 1e-3 with SCS < 2.0 and 5e-1 with SCS 2.0
-# linear2test needs 1e-4
-const config = MOIT.TestConfig(atol=1e-4, rtol=1e-4)
+    # linear9test needs 1e-3 with SCS < 2.0 and 5e-1 with SCS 2.0
+    # linear2test needs 1e-4
+    config = MOIT.TestConfig(atol=1e-4, rtol=1e-4)
 
-@testset "Continuous linear problems" begin
-    # AlmostSuccess for linear9 with SCS 2
-    MOIT.contlineartest(MOIB.SplitInterval{Float64}(optimizer), config, ["linear9"])
-end
+    @testset "Continuous linear problems" begin
+        # AlmostSuccess for linear9 with SCS 2
+        MOIT.contlineartest(MOIB.SplitInterval{Float64}(optimizer), config, ["linear9"])
```

Comment:
Is there a reasonable parameter setting that gets `linear9` passing?

Reply:
We need to set tolerance to 5e-1 last time I tested but it seems a bit restrictive to set it for all tests.
`linear9test(optimizer, MOIT.TestConfig(atol=0.5, rtol=0.5)` could be run separately in the line below

Reply:
```julia
 SCS.Optimizer(linear_solver=T, eps=1e-8))
 config = MOIT.TestConfig(atol=1e-5)
```
passes `MOIT.contlineartest(MOIB.SplitInterval{Float64}(optimizer), config)` with both `Direct` and `Indirect` solver;

btw. You need to add some tests for passing options to `SCS.Optimizer`. only playing with this allowed me to catch missing `;` in the argument list.

Reply:
I think I just ran into the same issue where `SCS.Optimizer` does not accept solver options.